### PR TITLE
Revert "chore(aws-lambda): removed timeout detection"

### PR DESCRIPTION
**Pull request by kirrg001**
Reverts https://github.com/instana/nodejs/pull/668

Brining this back but we make it optional.
Customer need this feature for debugging purposes.
We will also update the docs as well to make it clear

### CODE REVIEW

**Comment by kirrg001** [comment]
We will merge this as feature commit.

File: packages/serverless/test/util/AbstractServerlessControl.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const path = require('path');
 const fetch = require('node-fetch-v2');
 
 const retry = require('@instana/core/test/test_util/retry');
-const delay = require('@instana/core/test/test_util/delay');
+// const delay = require('@instana/core/test/test_util/delay');
```
Comment (aryamohanan) [inline packages/serverless/test/util/AbstractServerlessControl.js:16]: Instead of commenting, please remove this as we no longer use the delay.

Comment (kirrg001) [reply packages/serverless/test/util/AbstractServerlessControl.js:16]: Yes will do. I was waiting for the tests to pass on CI :)

Comment (kirrg001) [reply packages/serverless/test/util/AbstractServerlessControl.js:16]: There was a weird timeout for the backend stub for 2s, no idea why.

**Review by aryamohanan — APPROVED** [review]

**Pull request merged**